libs/commons/Testutil.ml

```diff
@@ -21,12 +22,14 @@ let list_map f l = List.rev_map f l |> List.rev
 let list_flatten ll =
   List.fold_left (fun acc l -> List.rev_append l acc) [] ll |> List.rev
 
-let pack_tests suite_name (tests : test list) : test list =
+let pack_tests suite_name (tests : _ list) : _ list =
```

Comment:
what does `_` do in this instance?

Reply:
It's effectively the analogue of `_` as a pattern in pattern matching. It means "there is a type here, which I do not care to write out", and is often used as a placeholder for the real type it corresponds to.

In this case, this is because I needed the function to work for both `test list` and `lwt_test` list. I could explicitly write it out as `'a test`, but in this case I was lazy and found it to be less verbose to just wildcard it out.

Reply:
since this code is touched by non-pdx, maybe it's worth explicitly defining a type of test = | Test | Lwt_test or something? Maybe not, I don't have strong feelings